usaspending_api/common/cache_decorator.py

```diff
@@ -2,11 +2,18 @@
 import logging
 from rest_framework_extensions.cache.decorators import CacheResponse
 
+from usaspending_api.common.experimental_api_flags import is_experimental_elasticsearch_api
+
 logger = logging.getLogger("console")
 
 
 class CustomCacheResponse(CacheResponse):
     def process_cache_response(self, view_instance, view_method, request, args, kwargs):
+        if is_experimental_elasticsearch_api(request):
+            # bypass cache altogether
+            response = view_method(view_instance, request, *args, **kwargs)
+            response = view_instance.finalize_response(request, response, *args, **kwargs)
```

Comment:
Are we sure we don't need to set `response["Cache-Trace"]` to something here? It's set, but not used, in the other code, so maybe it's being requested elsewhere in the program?

Reply:
What would you suggest we change it to? My understanding was that the "Cache-Trace" header is only set if the endpoint is actually interacting with the cache, which in this case it isn't.

Reply:
"no-cache" seems reasonable to me...

Reply:
That's what it does below when it checks but doesn't find anything

Reply:
@BrianZito But in the case where there is no cache decorator on an endpoint, this code isn't even encountered, and that header will never be set. 

That's the scenario this is intended to imitate:
- No even bothering with the cache

Rather than...

- i looked in the cache and didn't find anything

Reply:
I'm happy to see the code merged either way as  long as our  code  is never looking for the "header" (it's not a header, is it? It looks like a part of the response body).

Reply:
Cache-Trace is a header

Added to help devs and testers troubleshoot caching issues